Month in mathlib Oct and Nov (#61)
leanprover-community/blog: posts/month-in-mathlib-oct-nov-2022.md

    - [PR #13794](https://github.com/leanprover-community/mathlib/pull/13794) proves Schur's lemma.
    - [PR #17443](https://github.com/leanprover-community/mathlib/pull/17443) adds the construction of a projective resolution of a representation.

* Topology.
    - [PR #16677](https://github.com/leanprover-community/mathlib/pull/16677) constructs the Galois correspondence between closed ideals in $C(X, 𝕜)$ and open sets in $X$.
    - [PR #16719](https://github.com/leanprover-community/mathlib/pull/16719) shows that maximal ideals of $C(X, 𝕜)$ correspond to (complements of) singletons.
    - [PR #16087](https://github.com/leanprover-community/mathlib/pull/16087) defines covering spaces.
    - [PR #16797](https://github.com/leanprover-community/mathlib/pull/16797) proves that the stalk functor preserves monomorphism.
    - [PR #17015](https://github.com/leanprover-community/mathlib/pull/17015) proves that Noetherian spaces have finite irreducible components.

* Probability theory.
    - [PR #16882](https://github.com/leanprover-community/mathlib/pull/16882) proves the second Borel-Cantelli lemma.

>     - [PR #16648](https://github.com/leanprover-community/mathlib/pull/16648) show Kolmogorov's 0-1 law.

Reviewer: ```suggestion
    - [PR #16648](https://github.com/leanprover-community/mathlib/pull/16648) shows Kolmogorov's 0-1 law.
```

* Linear algebra.
    - [PR #11468](https://github.com/leanprover-community/mathlib/pull/11468) shows that the clifford algebra is isomorphic as a module to the exterior algebra.
    - [PR #16150](https://github.com/leanprover-community/mathlib/pull/16150) proves that the inverse of a block triangular matrix is block triangular.

* Category theory.
    - [PR #16969](https://github.com/leanprover-community/mathlib/pull/16969) adds basic results about localization of categories.

* Combinatorics
    - [PR #16195](https://github.com/leanprover-community/mathlib/pull/16195) adds the definition and some basic results about semistandard Young tableaux.
    - [PR #17445](https://github.com/leanprover-community/mathlib/pull/17445) combinatorics/young: add equivalence between Young diagrams and lists of natural numbers.

* Tactics
    - [PR #16313](https://github.com/leanprover-community/mathlib/pull/16313) introduces the `qify` tactic, to move from $\mathbb{Z}$ to $\mathbb{Q}$.
    - [PR #13483](https://github.com/leanprover-community/mathlib/pull/13483) adds a tactic for moving around summands.

>     - [PR #16911](https://github.com/leanprover-community/mathlib/pull/16911) adds a tactic to find the declarations use `sorry`, this is useful in large project.

Reviewer: ```suggestion
    - [PR #16911](https://github.com/leanprover-community/mathlib/pull/16911) adds a tactic to find declarations that use `sorry`. This tactic is intended for projects that depend on mathlib.
```
